app/assets/stylesheets/modules/facets.scss

```diff
@@ -64,4 +65,21 @@
       }
     }
   }
+
+  li.h-leaf {
+    margin-left: 15px;
+  }
+}
+
+.facet-hierarchy .facet-count {
+    display: inline-block;
```

Comment:
I'm seeing some 4 space indention here, where most other places we have 2 space. Would you mind fixing that?